Documentation/generics.md

```diff
@@ -0,0 +1,64 @@
+*.NET Innovation Examples*
+**GENERICS**
+===================
+
+We use generics implicitly all the time in  C#. When you use LINQ in C#, do you notice you are working with <code>IEnumerable\<T\></code>? Ever wonder what <code>T</code> is and what it means?
```

Comment:
We should decide on a good pattern for the title that we can use for the other documents you have planned. I need to think about that a bit. I'm hesitant to pivot on "innovation". Certainly, we built many features that are innovations, however, we'll want the ability to document a feature that doesn't cleanly fit as an "innovation" . It will be an exercise for the reader on which ones were truly innovative at the time they were released.

How about:

.NET Features: Generics
====================